kokonet-lab.yml

```diff
@@ -0,0 +1,65 @@
+- hosts: master, node, kokonet-vpp, kokonet-mgr
+  become: true
+  become_user: root
+  tasks:
+    - name: Install docker-py 
+      yum:
+        name: python-docker-py
+        state: latest
+  roles:
+    - { role: install-docker, when: container_runtime == 'docker' }
+
+- hosts: kokonet-mgr
+  tasks: 
+    - name: Run etcd container
+      docker_container:
+        name: etcd
+        image: quay.io/coreos/etcd:latest
+        command: ["etcd", "--advertise-client-urls",
+                  "'http://{{hostvars[inventory_hostname]['ansible_eth0']['ipv4']['address']}}:2379'",
+                  "--listen-client-urls", "'http://0.0.0.0:2379'"]
+        published_ports:
+          - 2379:2379
+    - name: Run kokonet-mgr container
+      docker_container:
+        name: nmgr
+        image: s1061123/kokonet:kokonet-mgr
+        env: 
+          VPP_CONTROLLER_URL: "http://{{hostvars['kokonet-vpp']['ansible_eth0']['ipv4']['address']}}:5000"
+          VPP_VTEP_IP: 10.1.1.1
+          ETCD_CLIENT_HOST: "{{hostvars[inventory_hostname]['ansible_eth0']['ipv4']['address']}}"
+          ETCD_CLIENT_PORT: 2379
+        published_ports:
+          - 80:80
+
+- hosts: kokonet-vpp
+  tasks: []
+  roles:
+    - { role: install-vpp }
+
+- hosts: kokonet-vpp
+  tasks: 
+    - name: Run vpp container
+      docker_container:
+        name: vpp
+        image: s1061123/kokonet:kokonet-vpp
+        privileged: true
+        volumes:
+          - /run/vpp:/run/vpp
+        published_ports:
+          - 5000:5000
+
+# kokonet/issue/6 will removes it.
```

Comment:
Might be worth it to just link directly to the issue for info?

Reply:
I don't think so because just a memo to identify and it must be short-live issue.